app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -16,7 +16,12 @@ def connect(options = {})
     key = authentication_key(options[:auth_type])
     region = options[:provider_region] || provider_region
     sdk = self.class.raw_connect(key)
-    sdk.vpc(:region => region)
+    if options[:service] == 'events'
+      service_key = authentication_key("events")
+      sdk.events(:region => region, :service_key => service_key)
```

Comment:
So if you need the provider region here, you can get that by doing `sdk.events(:region => region, :service_key => service_key, :provider_region => provider_region)` since its an attribute of the ext_management_system